source/locker/contracts/Locker.sol

```diff
@@ -110,32 +107,31 @@ contract Locker is Ownable {
    * @param account to lock tokens for
    * @param amount of tokens to lock
    */
-  function lockFor(address account, uint256 amount) public {
+  function lockFor(address account, uint256 amount) external {
     _lock(msg.sender, account, amount);
   }
 
   /**
    * @notice Unlock and transfer to msg.sender
    * @param amount of tokens to unlock
    */
-  function unlock(uint256 amount) public {
+  function unlock(uint256 amount) external {
     uint256 previous = withdrawals[msg.sender][epoch()];
 
     // Only enforce percentage above a certain balance
     if (balances[msg.sender] > throttlingBalance) {
       require(
         (previous.add(amount)) <=
-          ((throttlingPercentage.mul(balances[msg.sender])).div(100)),
+          (
+            (throttlingPercentage.mul(balances[msg.sender])).div(MAX_PERCENTAGE)
```

Comment:
This can become 

`throttlingPercentage.mul(balances[msg.sender]).div(MAX_PERCENTAGE)` (remove parenthesis)